packages/components/spacings/spacings-inline/src/types.ts

```diff
@@ -0,0 +1,27 @@
+// @TODO extract this into a shared file
```

Comment:
This is something we can do now or follow up. it's not priority IMO.
My concern with `spacings` is that we have redundant requirements.

Reply:
We should have the types within the same component file, otherwise the prop types generation won't work (babel doesn't know about other files when compiling this file).

Reply:
The thing about this is that this is also needed in `inline.styles` file.
Should I then import it there from the this component file?

Reply:
Yes